rebasehelper/checkers/abipkgdiff_tool.py

```diff
@@ -113,7 +113,7 @@ def run_check(cls, results_dir):
             logger.debug('Package name for ABI comparison %s', old_name)
             output = os.path.join(cls.results_dir, results_dir, old_name + '-' + cls.log_name)
             try:
-                ret_code = ProcessHelper.run_subprocess(command, output_file=output)
+                ret_code = ProcessHelper.run_subprocess(command, output=output)
```

Comment:
You are introducing a bug here and fixing it again in https://github.com/rebase-helper/rebase-helper/pull/438/commits/76b8fdc75ce75a87ccf205b431efc3fd97028b11.

Reply:
This still applies (except the commit hash with the fix is now d6d0c24e4855149c4184c8638e7f2539dbb3e247).

Reply:
Bumping this, as it shows as outdated.